src/openforms/registrations/contrib/objects_api/submission_registration.py

```diff
@@ -510,8 +515,7 @@ def get_update_payment_status_data(
                 submission=submission,
                 variables_registry=variables_registry,
             )
-            if variable.key
-            in ["payment_completed", "payment_amount", "payment_public_order_ids"]
+            if variable.key in PAYMENT_VARIABLE_NAMES
```

Comment:
hrm, this is a bit of an odd one in this form because the names of the variables are defined in `src/openforms/registrations/contrib/objects_api/registration_variables.py` but the constant is defined in the `payments` app. I'd expect the constant do be defined in the same file.